Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM adoptopenjdk
+EXPOSE 8880
+ARG JAVA_OPTS
+ENV JAVA_OPTS=$JAVA_OPTS
+RUN mkdir /opt/app
+WORKDIR /opt/app
+COPY ./build/libs/JSettlersServer-2.3.00.jar /opt/app/japp.jar
```

Comment:
For `COPY` and `RUN`, i'd prefer to keep the jsettlers name instead of generic japp.jar.
/opt/app/JSettlersServer.jar for example